contrib/native/client/CMakeLists.txt

```diff
@@ -22,6 +22,14 @@ cmake_policy(SET CMP0043 NEW)
 cmake_policy(SET CMP0048 NEW)
 enable_testing()
 
+if (CMAKE_VERSION VERSION_LESS "3.1")
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
```

Comment:
What about `MSVC` and others?

Reply:
Changed the approach to rising the min version as it required by protobuf.